### PR TITLE
Improve Python wrapping

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

**Comment by thewtex** [comment]
CI should be fixed in `master`

**Comment by fbudin69500** [comment]
@thewtex CI fails for ITK.Linux.Python. Could it be that the build takes too long?

File: CMakeLists.txt [hunk 1]
```diff
@@ -417,6 +417,12 @@ mark_as_advanced(ITK_COMPUTER_MEMORY_SIZE)
 option(ITK_USE_FLOAT_SPACE_PRECISION "Use single precision for origin/spacing/directions in itk::Image" OFF)
 mark_as_advanced(ITK_USE_FLOAT_SPACE_PRECISION)
 
+# This flag allows to no build itkTestDrivers while still configuring all the tests.
```
Comment (dzenanz) [inline CMakeLists.txt]: `to no build` -> `to not build`

**Comment by fbudin69500** [comment]
@thewtex : Now it is just under 5h with Python ON on Linux on CI!

**Pull request merged**